packages/storage-rn/package.json

```diff
@@ -50,14 +50,14 @@
   },
   "peerDependencies": {
     "@urql/exchange-graphcache": ">=5.0.0",
-    "@react-native-async-storage/async-storage": "^1.15.5",
-    "@react-native-community/netinfo": "^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0"
+    "@react-native-async-storage/async-storage": "^1.21.0",
```

Comment:
I don't think this is a fair bump as we still support 1.15.5

Reply:
very true! let me change that back!